pkg/services/accesscontrol/roles_test.go

```diff
@@ -9,26 +9,28 @@ import (
 )
 
 func TestPredefinedRoles(t *testing.T) {
-	for name, r := range FixedRoles {
+	for name, role := range FixedRoles {
```

Comment:
I think we can remove these tests since we decentralize the description, they don't make sense anymore.